wgsl/index.bs

```diff
@@ -6767,6 +6865,41 @@ The location of a function call is referred to as a <dfn noexport>call site</dfn
 Call sites are a [=dynamic context=].
 As such, the same textual location may represent multiple call sites.
 
+## Creation-time Functions ## {#creation-time-funcs}
+
+Function declarations annotated with the [=attribute/const=] attribute can be
+evaluated at [=shader module creation|shader-creation time=].
```

Comment:
The noun here is the declaration not the function, so this reads oddly: it's the call that's evaluated at creation time, not the declaration.

Suggest:

> A function declared with a with a [=attribute/const=] attribute can be evaluated at ...